### PR TITLE
README updated with V2 Compatibility

### PR DESCRIPTION
**Pull request by megabyte0x**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,11 +20,9 @@ Install ethers.js and Aarc SDK using npm:
 npm install ethers@5.7.2 aarc-sdk
 ```
 
-# Getting Started
-
 ## Get the API Key
 
-To use Aarc SDK, an API key is required. Fill out [this form](https://rebrand.ly/aarc-api) to get the API Key on your email instantly!
+To use Aarc SDK, an API key is required. Get the **API Key** from the [Dashboard](https://dashboard.aarc.xyz/).
```
Comment (anshulforyou) [inline README.md:25]: Mention the link of the gitbook as well here on the steps to fuel the gas tank

File: README.md [hunk 2]
```diff
@@ -52,7 +66,8 @@ let balances = await aarcSDK.fetchBalances(
 );
 ```
 
-## Moving Assets
+
+## Mirgate Assets
```
Comment (anshulforyou) [inline README.md:70]: Migrate spelling mistake

File: README.md [hunk 3]
```diff
@@ -73,23 +88,23 @@ await aarcSDK.executeMigration({
...
+  message: string;
+  txHash?: string;
+  tokenId?: string;
   },
-  ...
 ]
 ```
 
-### Moving Assets without gas
+## Migration Assets [With Gasless Flow]
```
Comment (anshulforyou) [inline README.md:105]: Migration assets sounds wrong, keep it migrate assets

File: README.md [hunk 4]
```diff
@@ -103,35 +118,48 @@ await aarcSDK.executeMigrationGasless({
       tokenIds?: string[] // tokenIds for nfts
     },
     ...
-  ],
-  gelatoApiKey: GELATO_RELAYER_API_KEY // Use the link above to get the gelato relayer key
+  ]
```
Comment (anshulforyou) [inline README.md:121]: Change the amount field as well. We now accept hex as string in amount instead of BigNumber

**Review by anshulforyou — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (megabyte0x) [reply README.md:25]: I have added the link and a single image of dashboard. Mentioning all the steps here is not necessary IMO.

File: README.md [hunk 5]
```diff
@@ -23,6 +23,12 @@ npm install ethers@5.7.2 aarc-sdk
 ## Get the API Key
 
 To use Aarc SDK, an API key is required. Get the **API Key** from the [Dashboard](https://dashboard.aarc.xyz/).
+You can learn about gettign the API Key from [here](https://docs.aarc.xyz/developer-docs/integration-guide/setup-and-installation).
```
Comment (anshulforyou) [inline README.md:26]: spelling mistake

File: README.md [hunk 6]
```diff
@@ -94,15 +112,15 @@ await aarcSDK.executeMigration({
   {
   tokenAddress: string;
   taskId?: string;
-  amount?: BigNumber | BigNumberish;
+  amount?:string
```
Comment (anshulforyou) [inline README.md:115]: ; missing

File: README.md [hunk 7]
```diff
@@ -129,15 +147,15 @@ await aarcSDK.executeMigrationGasless({
...
   message: string;
   txHash?: string;
   tokenId?: string;
   },
 ]
 ```
 
-## Migration of Assets [Pay Gas with Stables]
+## Migrate of Assets [Pay Gas with Stables]
```
Comment (anshulforyou) [inline README.md:158]: It should be Migration of Assets

**Review by anshulforyou — CHANGES_REQUESTED** [review]

**Pull request closed without merging**